Source/Modules/csharp.cxx

```diff
@@ -46,6 +47,8 @@ class CSHARP:public Language {
   bool global_variable_flag;	// Flag for when wrapping a global variable
   bool old_variable_names;	// Flag for old style variable names in the intermediary class
   bool generate_property_declaration_flag;	// Flag for generating properties
+  bool doxygen;			//flag enabling comment processing
+  bool comment_creation_chatter; //flag for getting information about where comments were created
```

Comment:
Was this supposed to be left in? Because there doesn't seem to be any way to set it, so it doesn't look useful in its current form.

Reply:
it is also no way to set it in Java module, but it may be set at runtime while debugging

Reply:
Maybe set from the debugger (just like in Java module)